lib/workers/repository/update/branch/index.ts

```diff
@@ -209,9 +209,15 @@ export async function processBranch(
         };
       }
     }
+
+    logger.debug({
```

Comment:
Are you sure you want this printed once per branch, even if no limits are used? Maybe use a more compact form without metadata

Reply:
> Before executing branches, we should calculate the hourly and concurrent existing and log these values (i.e. log how many are existing, not how many left)

I misunderstod this and thought we need to log before each branch.
I will move it above in the flow.